erts/emulator/beam/erl_bits.c

```diff
@@ -78,6 +78,11 @@ typedef Uint16 erlfp16;
 
 static byte get_bit(byte b, size_t a_offs); 
 
+/* If the proc bin is larger than 64 MB,
+   we only increase by 10% instead of doubling */
+#define NEW_PROC_BIN_SIZE(size) \
+    ((size) > (1ull << 26)) ? 1.1*(size) : 2*(size)
+
```

Comment:
Best to put parentheses around the entire macro.

and naming is hard... "NEW" gives me the wrong associations, may I suggest something like GROW_PROC_BIN_SIZE.

To me 64MB sounds a bit large. What about 8MB and 20%

Reply:
... or PROC_BIN_ALLOC_SIZE or PROC_BIN_CAPACITY?

Reply:
I did 16MB and 20% as a compromise :)